invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.variables

```diff
@@ -7,3 +7,5 @@
 // Tablets/phones were left as-is because they have less screen estate
 @selectionComputerMaxMenuHeight : 575px;
 @selectionWidescreenMaxMenuHeight : 575px;
+
+@membersContainerPadding: @selectionVerticalPadding @selectionHorizontalPadding @selectionVerticalPadding @selectionHorizontalPadding;
```

Comment:
On second thought, this might be unnecessary as it is too specific for members table and not that re-usable. What do you think?

Reply:
I think maybe not needed as a new variable, you could assign straight to the class:
```less
.members-action-status-container { 
padding: @selectionVerticalPadding @selectionHorizontalPadding;
```
PS. no need to assign the variable for all 4 directions, if you have
```
padding: 1em 2em;
```
it assigns the paddings per "axis" and shortens the notation

Reply:
thanks for the hints! 👍